src/App.jsx

```diff
@@ -9,32 +11,52 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState('tab-1');
```

Comment:
```suggestion
  const [selectedTabId, set
  selectedTabId] = useState(tabs[0].id);
```
better to set id from first tabs element
consider more meaningful naming